packages/menu/src/MenuGroup.ts

```diff
@@ -41,13 +46,38 @@ export class MenuGroup extends SpectrumElement {
         MenuGroup.instances += 1;
     }
 
+    private get menuRole() {
+        if (this.selects) {
+            return 'menu';
+        } else {
+            return 'presentation';
+        }
+    }
+
+    // Menu defaults to no selection, while MenuGroup defaults to inherit.
+    private get menuSelects(): void | String {
+        switch (this.selects) {
+            case undefined:
+                return 'inherit';
+            case 'none':
+                return undefined;
+            default:
+                return this.selects;
+        }
+    }
+
     public render(): TemplateResult {
         const labelledby = `menu-heading-category-${this.instanceCount}`;
         return html`
             <span class="header" id=${labelledby} aria-hidden="true">
                 <slot name="header"></slot>
             </span>
-            <sp-menu role="presentation">
+            <sp-menu
+                selects=${ifDefined(
+                    this.menuSelects ? this.menuSelects : undefined
```

Comment:
Should we just have `menuSelects` return `undefined` to simplify the templating here?